src/main/java/Task.java

```diff
@@ -0,0 +1,42 @@
+public class Task {
+	private boolean isDone;
+	private String taskContent;
+
+	public Task(String taskContent) {
+		this.taskContent = taskContent;
+		this.isDone = false;
+	}
+	public void markAsDone() {
+		this.isDone = true;
+	}
+	public void test() throws DukeException {
+		if (taskContent.length() == 0) {
+			throw new DukeException("☹ OOPS!!! The description of a task cannot be empty.");
+		}
+	}
+	public String getType() {
+		return "";
+	}
+	public boolean isDone() {
+		return this.isDone;
+	}
+	public String  getContent() {
+		return this.taskContent;
+	}
+	public String  getDate() {
+		return "";
+	}
+	@Override
+	public String toString() {
+		if (isDone) {
+			return "[" + "\u2713" + "] " + this.taskContent; 
+		} else {
+			return "[" + "\u2718" + "] " + this.taskContent;
+		}
+		// if (isDone) {
```

Comment:
Perhaps it might be cleaner if you delete dead code instead of commenting them out? Since you could always retrieve old code from previous commits.